CHANGELOG.markdown

```diff
@@ -1,7 +1,8 @@
-next [????.??.??]
+4.19 [????.??.??]
```

Comment:
Go ahead and bump the version in `lens.cabal`, while you're at it.

Reply:
Done